vllm/_core_ext.py

```diff
@@ -0,0 +1,84 @@
+import importlib.util
+from typing import TYPE_CHECKING
+
+import torch
+
+from vllm.logger import init_logger
+
+logger = init_logger(__name__)
+core_C_available = importlib.util.find_spec('._core_C', 'vllm') is not None
+
+if core_C_available:
+    try:
+        # ruff: noqa: F401
+        import vllm._core_C
```

Comment:
I think this disables the check for the entire file.  To suppress just a single line it should come at the end, e.g.  `import vllm._core_C # noqa: F401`

Reply:
good to know, thanks done